models/gc_grt123_lung_cancer/meta.json

```diff
@@ -0,0 +1,144 @@
+      "code": "2023-07-04",
+      "pub": "2017-11-22"
+    },
+    "cite": "Liao F, Liang M, Li Z, Hu X, Song S. Evaluate the Malignancy of Pulmonary Nodules Using the 3-D Deep Leaky Noisy-OR Network. IEEE Trans Neural Netw Learning Syst. 2019;30(11):3484-3495.",
+    "license": {
+      "code": "MIT",
+      "weights": ""
+    },
+    "publications": [
+      {
+        "title": "JOURNAL OF LATEX CLASS FILES, VOL. 14, NO. 8, AUGUST 2015 1 Evaluate the Malignancy of Pulmonary Nodules Using the 3D Deep Leaky Noisy-or Network",
+        "uri": "https://arxiv.org/pdf/1711.08324.pdf"
+      },
+      {
+        "title": "Deep Learning for Lung Cancer Detection on Screening CT Scans: Results of a Large-Scale Public Competition and an Observer Study with 11 Radiologists",
+        "uri": "https://pubmed.ncbi.nlm.nih.gov/34870218/"
+      }
+    ],
+    "github": "https://github.com/DIAGNijmegen/bodyct-dsb2017-grt123",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "This algorithm analyzes non-contrast CT scans of the thorax and predicts the lung cancer risk.",
+      "references": [
+        {
+          "label": "Lung cancer risk estimation algorithm on grand-challenge",
+          "uri": "https://grand-challenge.org/algorithms/dsb2017-grt123/"
+        }
+      ],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "The nodule detection was evaluated on the validation set of Data Science Bowl 2017 challenge. It contains data from 198 cases and there are 71 (7 nodules smaller than 6 mm are ruled out) nodules in total. The Free Response Operating Characteristic (FROC) is used to evaluate the performance and can be viewed in the publication. The Case classification is evaluated using the Area Under the Curve (AUC) metric.",
+      "references": [{
+        "label": "Data Science Bowl 2017 challenge",
+        "uri": "https://www.kaggle.com/c/data-science-bowl-2017"
+      }],
+      "tables": [{
+        "label": "AUC for the Case classification",
+        "entries": {
+          "AUC training set": "0.90",
+          "AUC test set": "0.87"
+        }
+      }]
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The model was evaluated against a private dataset of 300 low-dose CT images, containing 100 cancer-positive scans and 200 cancer-negative scans.",
+      "references": [
+        {
+          "label": "Evaluation paper",
+          "uri": "https://pubmed.ncbi.nlm.nih.gov/34870218/"
+        }
+      ],
+      "tables": []
+    },
+    "training": {
+      "title": "Training data",
+      "text": "The Luna dataset includes the images from the LIDC/IDRI dataset in a different format, together with additional annotations. The LIDC/IDRI dataset is available at https://wiki.cancerimagingarchive.net/display/Public/LIDC-IDRI under a Creative Commons Attribution 3.0 Unported License.",
```

Comment:
Please use references for all links and write [n] to 'reference' the nth reference.

Reply:
This description suggests that all data for training comes from https://wiki.cancerimagingarchive.net/display/Public/LIDC-IDRI. 

1. The link is broken but offers a [follow-up](https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=1966254) link. Please us the correct link instead.
2. The website found under the follow-up link reports `1018 cases`. The `LIDC-IDRI_MetaData` csv contains 1308 rows. You report 2285 training cases. It's absolutely not clear to me on what data the model was trained on. Most importantly, the composition and potential selection criteria (if applied) are unclear.

Reply:
You are right, this isn't very clear. The training data consists of:
1. The LuNa dataset. This is a subset of the LIDC-IDRI dataset, excluding scans with a slice thickness greater than 2.5 mm, resulting in a total of 888 CT scans. I can change the link to the dataset of the Luna challenge if that would make things clearer.
2. The Kaggle dataset, which contains `1397` cases in it's training set. This is why there is a link to the dataset, eventhough it's not available anymore. 

> The LIDC-IDRI_MetaData csv contains 1308 rows.

True, but it contains different modalities. When you select only CT, there are `1018` entries, which matches the number of cases reported.

Reply:
Okay, great, then just update the text with this information. That the dataset is no longer available is no issue per se; please highlight it in the text though, so that we set the expectations right.

Reply:
We updated the information and the links.